multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -508,6 +509,26 @@ def column_types(self):
         else:
             return self._column_types
 
+    @property
+    def total_parameters(self) -> int:
+        return sum(p.numel() if not is_lazy_weight_tensor(p) else 0 for p in self._model.parameters())
+
+    @property
+    def trainable_parameters(self) -> int:
+        return sum(
+            p.numel() if not is_lazy_weight_tensor(p) else 0 for p in self._model.parameters() if p.requires_grad
+        )
+
+    @property
+    def model_size(self) -> float:
+        precision_to_bits = {64: 64, 32: 32, 16: 16, "bf16": 16}
+        if self._config is not None:
+            precision = precision_to_bits.get(self._config.env.precision, 16)
```

Comment:
wondering how mixed precision can be handled here? Shall we use `total_size_in_bytes = sum(p.numel() * p.element_size() for p in  self._model.parameters())` to address it?

Reply:
Good suggestion. Updated.